src/apiCall.js

```diff
@@ -1,8 +1,8 @@
 const _ = require('lodash')
 const objectPath = require('object-path')
-const { expect } = require('chai');
+const { expect } = require('chai')
 
-const { loadYamlFile } = require('./util.js')
+const { loadYamlFile, evaluateFaker, fakerScopes } = require('./util.js')
```

Comment:
Util is starting to look like a dumping ground for all sorts of stuff. This is actually a pretty common issue with anything called "util". Maybe split the functionality into smaller files and if you still want to, do an index.js in a util folder that will pull in all the bits.

Reply:
I agree I splited this to another class, we better have a specific implementation for  our custom types